src/model_config.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018-2020, NVIDIA CORPORATION. All rights reserved.
+// Copyright (c) 2018-2022, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Sorry for missing this spot and not being clear, the new format for copyright is the one in model_config.h
```
// Copyright 2018-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```